documentation/src/main/markdown/releasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Release notes
 
+## Version 8.7.0
+
+### Resolved issues
```

Comment:
Hi Shanty, two minor change requests please: For "Detect" if you could use the variable _[solution_name]_ and perhaps this would go under a "Changed features" heading instead of Resolved issues.
Thanks!